README.md

```diff
@@ -63,6 +63,12 @@ an Active Directory domain controller's hostname (do not use IP address) may be
 
 Default: Not set
 
+#### ad_integration_force_rejoin
+
+Leave existing domain prior to performing join.
```

Comment:
It would be nice to include some use cases for this

Reply:
Added the main use case.